README.md

```diff
@@ -7,27 +7,77 @@ test suite, Tempest.
 
 The tempest snap can be installed directly from the snap store:
 
-    sudo snap install --edge tempest
+    sudo snap install [install-OPTIONS] tempest
+
+## Plugins
+
+The snap provides all the plugins listed in the [OpenStack release notes] and those
+cannot be removed or updated. Currently, it is not possible to add new plugins or use
+your own.
+
+List of plugins:
+
+- barbican-tempest-plugin
+- blazar-tempest-plugin
+- cinder-tempest-plugin
+- cloudkitty-tempest-plugin
+- cyborg-tempest-plugin
+- designate-tempest-plugin
+- ec2api-tempest-plugin
+- freezer-tempest-plugin
+- glance-tempest-plugin
+- heat-tempest-plugin
+- ironic-tempest-plugin
+- keystone-tempest-plugin
+- kuryr-tempest-plugin
+- magnum-tempest-plugin
+- manila-tempest-plugin
+- mistral-tempest-tests
+- monasca-tempest-plugin
+- murano-tempest-plugin
+- neutron-tempest-plugin
+- octavia-tempest-plugin
+- oswin-tempest-plugin
+- sahara-tests
+- senlin-tempest-plugin
+- solum-tempest-plugin
+- telemetry-tempest-plugin
+- trove-tempest-plugin
+- venus-tempest-plugin
+- vitrage-tempest-plugin
+- watcher-tempest-plugin
+- zaqar-tempest-plugin
+- zun-tempest-plugin
 
 
 ## Quickstart
-To run Tempest, please follow the steps in [Tempest QuickStart](https://docs.openstack.org/tempest/latest/overview.html#quickstart) documentation.
+To run Tempest, please follow the steps in the [Tempest QuickStart] documentation.
 
-The tempest snap provides a set of pre-defined test lists with different focuses. For quickly verifying the behavior of the objective cloud and making sure tempest is correctly set up, you can run tests listed in the `readonly-quick.txt` file:
+The tempest snap provides a set of pre-defined test lists with different focuses.
```

Comment:
well, only one for now...

Reply:
As discussed elsewhere, we are keeping this wording plural because more test lists are going to be added in the following PRs.